lib/tasks/database.rake

```diff
@@ -16,6 +16,7 @@ namespace 'db:sessions' do
     cutoff_period = (ENV['SESSION_DAYS_TRIM_THRESHOLD'] || 30).to_i.days.ago
     ActiveRecord::SessionStore::Session.
       where("updated_at < ?", cutoff_period).
```

Comment:
Nice! However, even this solution gave me a timeout, but at least it deleted many batches instead of nothing

what i'm gonna try is add some throttling between each batch (like suggested in rails docs )
![image](https://github.com/user-attachments/assets/5d2e446d-81c8-4897-b0e0-c656753bd9ce)